documentation/suggested-project-structure.md

```diff
@@ -0,0 +1,97 @@
+### Suggested Project Structure
+
+ * `server.js` is the main entry point.
+ * Think of the `resources` folder as your `routes`.
+ * Think of the `handlers` folder as your `controllers`.
+
+```
+├── handlers
+│   ├── articleHandler.js
+│   ├── commentHandler.js
+│   ├── peopleHandler.js
+│   ├── photoHandler.js
+│   └── tagHandler.js
+├── resources
+│   ├── articles.js
+│   ├── comments.js
+│   ├── people.js
+│   ├── photos.js
+│   └── tags.js
+└── server.js          
+```
+
+#### Example server.js
+
+This is the main entry point for your API. The goal is to configure jsonapi-server and load all the resources.
+
+```javascript
+var jsonApi = require("jsonapi-server");
+var fs = require("fs");
+var path = require("path");
+
+jsonApi.setConfig({
+  // Put your config here!
+});
+
+// Load all the resources
+fs.readdirSync(path.join(__dirname, "/resources")).filter(function(filename) {
+  return /^[a-z].*\.js$/.test(filename);
+}).map(function(filename) {
+  return path.join(__dirname, "/resources/", filename);
+}).forEach(require);
+
+jsonApi.start();
+```
+
+#### Example resource
+
+The idea is to stick to having one resource per file and stick to pure config. Each file should do nothing more than define a resource. Handlers should be referenced from the `../handlers` folder, which enables us to easily abstract functionality by sharing features amongst handlers. Resource files are effectively defining your routing layer.
```

Comment:
it's not misleading (except perhaps the last part about routing), more convoluted. I suppose "what" a resource is is defined elsewhere though?